src/ngraph/runtime/plaidml/CMakeLists.txt

```diff
@@ -13,6 +13,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ******************************************************************************
+set(CMAKE_MACOSX_RPATH 1)
```

Comment:
Do we need to set this?
MACOSX_RPATH is already enabled here.
https://github.com/NervanaSystems/ngraph/blob/master/CMakeLists.txt#L63-L64